samples/angular-todo/src/app/app.module.ts

```diff
@@ -87,6 +90,9 @@ import { ChoiceComponent } from './features/journey/choice/choice.component';
     TodoIconComponent,
     ActionIconComponent,
     ChoiceComponent,
+    IdentityProviderComponent,
+    GoogleIconComponent,
+    AppleIconComponent,
```

Comment:
Are we missing Facebook? It's one of the three we officially support, so we should probably have an example of it.

Reply:
hi justin, facebook it is not here becasue if oyu have a look at the facebook button there's no icon to be added. It has no an "apple" or a "G" as apple and google does.

Reply:
Okay, if that's consistent with Facebook's display rules, then I'm good with it.

Reply:
https://developers.facebook.com/docs/facebook-login/userexperience#buttondesign

it seems like you can use either, maybe we should be consistent with the other buttons?